package.json

```
@@ -2,26 +2,48 @@
   "name": "zenodo",
   "version": "1.0.2",
   "description": "Node.js library to access the Zenodo API",
-  "main": "src/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "src"
+  ],
   "scripts": {
-    "test": "echo OK"
+    "build": "npm run tsc-esm && cheminfo-build --entry lib-esm/index.js --root Zenodo",
+    "check-types": "tsc --noEmit",
+    "clean": "rimraf lib lib-esm",
+    "eslint": "eslint src",
+    "eslint-fix": "npm run eslint -- --fix",
+    "prepack": "npm run tsc",
+    "prettier": "prettier --check src",
+    "prettier-write": "prettier --write src",
+    "test": "npm run test-coverage && npm run eslint && npm run prettier && npm run check-types",
+    "test-coverage": "vitest run --coverage",
+    "test-only": "vitest run",
```

It seems you copied the package.json from a project with our old standards.

```suggestion
    "test": "npm run test-only && npm run eslint && npm run prettier && npm run check-types",
    "test-only": "vitest run --coverage",
```